csharp/stringreversal.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+					
+public class Program
+{
+	public static void Main()
+	{
+		string line = Console.ReadLine();
+		
+		Func<string, string> fac = null; fac = n => (n.Length < 1) ? string.Empty : n[n.Length-1] + fac(n.Substring(0, n.Length - 1));
```

Comment:
```suggestion
		Func<string, string> fac = n => (n.Length < 1) ? string.Empty : n[n.Length-1] + fac(n.Substring(0, n.Length - 1));
```
I'm not familiar with C# but is this allowed? It would eliminate the need for using ; to separate two statements and make this a "one-liner"

Reply:
yes

Reply:
Edited the file, I guess if it's not allowed someone can submit a new issue about it